Assets/Scripts/BallScript.cs

```diff
@@ -44,23 +38,19 @@ public class BallScript : MonoBehaviour {
 	public GameObject circleTrail;
 	int lastScore;
 
-	public GameObject Arena1;
-	public GameObject Arena2;
-	public GameObject Arena3;
-	public GameObject Arena4;
+	// Ball options
+	public bool gravChangeMode = true;
```

Comment:
Ah cool! I hadn't thought of not wanting grav changes for some challenges. This'll be handy.